etc/export_latiss.yaml

```diff
@@ -0,0 +1,23 @@
+datasets:
+- collections: LATISS/templates
+  datasetType: goodSeeingCoadd
+- collections: refcats*
+  datasetType: atlas_refcat2_20220201
+- collections: refcats*
+  datasetType: gaia_dr3_20230707
+- collections: skymaps
+  datasetType: skyMap
+# Workaround: use a matching expression rather than a specific
+# string "LATISS/calib" for the collection argument, so to avoid
+# MissingCollectionError when the collection does not exist in
+# the target repo.
```

Comment:
Why do you need a workaround? `_filter_datasets` is supposed to be runnable on an empty destination repo...

Reply:
it doesn't if I constrain it for a specific collection and that collection doesn't exist in the destination repo.  It's an extra keyword arguments to [`dest_butler_repo.registry.queryDatasets(*args, **kwargs)`](https://github.com/lsst-dm/prompt_processing/blob/main/python/activator/middleware_interface.py#L1238)  and 
`queryDatasets(...,collections="something-not-in-repo")` gives `MissingCollectionError`.  

I guess usually we only use it with well-defined collection names and those names are registered in the empty local butler repo before any `_filter_datasets`?

Reply:
Yes, but we could have a handler similar to https://github.com/lsst-dm/prompt_processing/blob/main/python/activator/middleware_interface.py#L1239-L1242. Can you ticket it as a bug with a how-to-reproduce? It should be an easy fix.

Reply:
That'd be a great addition to `_filter_datasets` --> https://rubinobs.atlassian.net/browse/DM-43294